docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -158,88 +176,60 @@ Variables are part of a process instance and represent the data of the instance.
 
 <!-- TODO note that processes can be of any complexity, and link to advanced guides -->
 
-## Step 2: Design a form
+## Step 2: Design and link a form
 
-You have now designed the process. To allow the user to make the decision, you must next design a [form](../components/modeler/forms/camunda-forms-reference.md). Forms are added to user tasks and start events to capture user input, with the user input used to route the process flow, make API requests, or orchestrate your services.
+You have now designed the process. To allow the user to make the decision, you must next design a [form](../components/modeler/forms/camunda-forms-reference.md). Forms are added to user tasks and start events to capture user input that is used to route the process flow, make API requests, or orchestrate your services.
 
 <Tabs groupId="install" className="tabs-hidden">
 <TabItem value="saas">
 
-1. Select the user task you created in **[Step 1](#step-1-create-a-new-process)**.
-1. Click the blue **link icon** in the lower right corner. A menu expands that allows you to create a new form.
-
-   <p><img src={ModelerFormMenuImg} style={{width: 400}} alt="Annotation to open the form menu" /></p>
+:::note
 
-1. Click **Create new form**. A form is created and opened in the form editor. The form is automatically named.
+- You do not need to save your process diagram as Web Modeler automatically saves every change you make.
+- You can also create [milestones](/components/modeler/web-modeler/milestones.md) to save a version snapshot of your BPMN or DMN diagram at any time.
 
-   :::note
-   You do not need to save your process diagram as Web Modeler automatically saves every change you make.
-   :::
+:::
 
+1. Select the user task you created in **[Step 1](#step-1-create-a-new-process)**.
+1. Click the blue **Link Form** icon in the context pad, and click **Create new form**.
```

Comment:
This is the first time the "context pad" is mentioned, but there's nothing showing what it looks like or where to find it. The same thing applies to the palette. @lmbateman are these intuitive enough concepts that we can assume the user knows what we're talking about?

Reply:
I've added a link to the context pad page here 👍